src/controllers/projectController.ts

```diff
@@ -53,12 +53,11 @@ export class ProjectController {
         const projectFile: string = path.join(basePath, projectName, ".project");
         const extensionPath: string = this.context.extensionPath;
         try {
-            await Promise.all([
```

Comment:
Maybe ensure the directory first and put all the copy work in one Promise.all()?

Reply:
I agree with @SummerSun and disagree with the proposed solution. Promise.all is well known so it does not make things harder to understand. The performance downgrade is definitely noticeable when you work with larger projects. Triggering multiple copy task and let them run in parallel will make the most of modern hardware like SSD. Running them in serial manner does not make any sense.